packages/alea-frontend/pages/api/job-portal/create-invite-to-org.ts

```diff
@@ -0,0 +1,27 @@
+import { Action, ResourceName } from '@stex-react/utils';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getUserIdIfAuthorizedOrSetError } from '../access-control/resource-utils';
+import { checkIfPostOrSetError, executeAndEndSet500OnError } from '../comment-utils';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!checkIfPostOrSetError(req, res)) return;
+  const { orgId, email } = req.body;
+  if (!email || !orgId) return res.status(422).send('Email or OrgId missing');
+  const inviterId = await getUserIdIfAuthorizedOrSetError(
+    req,
+    res,
+    ResourceName.JOB_PORTAL_ORG,
+    Action.MANAGE_JOB_POSTS,
```

Comment:
This API is related to adding users ina org. The action should be `ACCESS_CONTROL`.